scripts/irods/test/test_federation.py

```diff
@@ -1777,3 +1777,134 @@ def test_local_zone_user_is_not_allowed_to_delete_delay_rules_created_by_remote_
 
         finally:
             self.local_admin.run_icommand(['iqdel', '-a'])
+
+
+class test_compound_resource_operations(SessionsMixin, unittest.TestCase):
```

Comment:
I'm not too attached to this name or even the idea of this being a separate class. I did it this way because of all the setup required for this to work.

Reply:
It's fine with the name as long as it gives the reader a decent idea about what to expect.

If the current name is not clear, please use more words.

Reply:
Okay, then I will just leave it. Thanks